carbon-design-system/carbon
# docs(examples): update codesandbox

It seems like our current codesandbox is hanging on installing `web-vitals`. This PR updates our example to no longer include this dependency. It also removes other superfluous dependencies that aren't used in the environment

#### Changelog

**New**

**Changed**

- Remove `web-vitals` dependency from codesandbox example
- Remove test dependencies from codesandbox example

**Removed**

#### Testing / Reviewing

- Pull down PR
- Run `yarn install`
- Run `yarn `start` in the `examples/codesandbox` directory
- Verify the button appears as expected
